packages/docs/docs/app/bots.md

```diff
@@ -44,98 +44,81 @@ console.log('Hello world');
 - Scroll down and click "OK"
 
-Now you have an active Subscription.
+Now you have an active Subscription. ([View all Subcriptions](https://app.medplum.com/Subscription))
 
 ## Execute the bot
 
-With our Bot and Subscription in place, let's trigger the Bot.
+With our [Bot](https://app.medplum.com/Bot) and [Subscription](https://app.medplum.com/Subscription) in place, let's trigger the Bot.
 
-- First, go to the Patient resources using the top-left menu and clicking "Patient"
-- Next, click on the "New..." button
-- Enter a sample name such as given "Bot" family "Demo"
+- First, go to the [Patient resources](https://app.medplum.com/Patient) using the top-left menu and clicking "Patient"
+- Next, click on the "New..." [button](https://app.medplum.com/Patient/new)
+- Enter a sample name such as given "Jane" family "Doe"
 - Scroll down and click "OK"
 
-Now, let's go back to our Subscription. On the Timeline, you should see an AuditEvent with the outcome of the JavaScript code execution. If everything worked as expected, you should see "Hello world" in the "outcomeDesc" property.
+Now, let's go back to our [Subscription](https://app.medplum.com/Subscription). On the Timeline, you should see an AuditEvent with the outcome of the JavaScript code execution. If everything worked as expected, you should see "Hello world" logged as part of the AuditEvent.  If you want to see all AuditEvents sorted by most recent, you can use [this link](https://app.medplum.com/AuditEvent?_count=20&_fields=id,_lastUpdated&_offset=0&_sort=-_lastUpdated).
 
 ## Bot sandbox
 
-The JavaScript code is heavily sandboxed for security reasons. Only a handful of variables are exposed:
+The JavaScript code is heavily sandboxed and runs in an AWS Lambda.  You can apply an [AccessPolicy](https://app.medplum.com/AccessPolicy) to the Bot if you want to further reduce the data it can read and write.
+
+The following resources are available to the Bot code, to enable it to do the functionality it requires.
 
 | Name              | Type     | Description                                                                       |
 | ----------------- | -------- | --------------------------------------------------------------------------------- |
-| `resource`        | object   | The FHIR resource that triggered the bot/subscription                             |
-| `console`         | object   | A normal console-like variable that can be used for logging output to AuditEvents |
-| `repo`            | object   | A FHIR repository helper. See below.                                              |
-| `assertOk`        | function | Helper function to validate the outcome of a repository call. See below.          |
-| `createReference` | function | Helper function to create FHIR References from FHIR Resources.                    |
+| `medplum`       | function   | An instance of the medplum JS SDK ([documentation](https://docs.medplum.com/typedoc/core/index.html))                             |
```

Comment:
The type is not `function`, it's `MedplumClient`

Reply:
Good point